### DISCUSSION TITLE
Happy Pride 🌈 🏳️‍🌈 🏳️‍⚧️ ❣️!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#58667] [Announcements]
We’re happy to be celebrating Pride in the GitHub Community and we’d love for you to join us 🏳️‍🌈. Whether it’s contributing to an LGBTQIA+ open source project or making your repo more rainbow - we’d love to know how you are celebrating! 

What is Pride? On June 28, 1969, queer New Yorkers rioted against police brutality outside the Stonewall Inn, one year later on the anniversary, a march commemorated the anniversary of the protest. Now Pride is celebrated in the month of June and is meant to celebrate, uplift, and support the rights of queer folks. For more on Pride, check out [this link](https://www.loc.gov/lgbt-pride-month/about/). 

⌨️ **Contribute to a project that supports the queer community**:
* [InReach](https://github.com/weareinreach/.github/blob/main/CONTRIBUTING.md) - The world’s first tech platform, matching LGBTQIA+ people with safe, verified resources
* [RefugeRestroom](https://github.com/RefugeRestrooms/refugerestrooms/wiki) - Indexes and maps safe restroom locations for trans, intersex, and gender nonconforming individuals

💟 **Deck out your coding setup for the festivities** ❤️ 🧡 💛 💚 💙 💜:
* [Pride flags for your terminal](https://github.com/ExperiBass/cli-pride-flags): Display various pride flags in your terminal
* [LGBT Pride in GitHub repository languages](https://github.com/spacekookie/gay): Make your languages rainbow 🌈
* [Microsoft’s Pride Designs](https://github.com/microsoft/Pride): At a time when Pride is increasingly harder to celebrate, Microsoft is empowering organizers worldwide by open-sourcing their designs and inviting everyone to remix, share, and create more Pride

:octocat: **Check out how GitHub is making open source more inclusive**: 
* **Pronouns on your profile** 🖼️: [Add them](https://github.blog/changelog/2023-03-01-add-pronouns-to-your-github-profile/), and now on [mobile](https://github.blog/changelog/2023-05-10-display-pronouns-on-github-mobile-profile/) too!
* **[Announcing the All In CHAOSS DEI Badging pilot initiative](https://github.blog/2023-06-07-announcing-the-all-in-chaoss-dei-badging-pilot-initiative/)**: The program allows open source maintainers to signal their ongoing efforts to improve and prioritize DEI within their communities and have the ability to recognize those who are leading the way.

🪴 **Resources**:
* [Map of queer resources](https://github.com/qiekub/map): A map of community centers and other helpful information for queer folks
* [Resources for queer folks in tech](https://github.com/ryderdamen/lgbtq_technology_resources): Check out the resources and submit your own! 

There is still plenty of work to be done around the world, in open source, and online for the LGBTQIA+ community, but GitHub maintains the belief that we should be contributing to an open, welcoming, and inclusive community 🌈. From us to you, wishing you a wonderful rest of your Pride month 🏳️‍🌈!

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
